utils/docker/run-bindings.sh

```diff
@@ -31,7 +31,7 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #
-# run-bindings.sh - checks bindings' building and installation
+# run-bindings.sh - checks bindings' building and installation (on tag 0.8)
```

Comment:
this is a redundant information which people will forget to update

Reply:
Done.